arcgis-java-toolkit/README.md

```diff
@@ -0,0 +1,108 @@
+# arcgis-java-toolkit
+
+## Introduction
+
+The ArcGIS Runtime SDK for Java Toolkit contains controls and utilities to simplify your app development. The toolkit is provided as an open source resource, so you can feel free to download or clone the code and customize to meet your requirements.
+
+## Features
+
+The latest version of the ArcGIS Runtime SDK for Java Toolkit features the following JavaFX components:
+
+- Compass: Shows the current viewpoint heading. Can be clicked to reorient the view to north.
+- Feature Template Picker: Shows feature templates for a collection of feature layers.
+- Floor Filter: Shows sites and facilities, and enables toggling the visibility of levels on floor aware maps and scenes.
+- Overview Map: Indicates the viewpoint of the main map/scene view.
+- Scalebar: Shows a ruler with units proportional to the map's current scale.
+
+## Requirements
+
+The toolkit requires the ArcGIS Runtime SDK for Java. Refer to the 'Instructions' section below if you are using Gradle.
+See [the developer guide](https://developers.arcgis.com/java/install-and-set-up/) for complete instructions and
+getting setup with the SDK.
+
+The following table shows the minimum version of the SDK compatible with the toolkit:
+
+| SDK Version | Toolkit Version |
+|-------------|-----------------|
+| 100.2.1     | 100.2.1         |
+| 100.14.0    | 100.14.0        |
+
+### API Key requirements
+
+Some of the toolkit components utilize ArcGIS Platform services which require an API key. Refer to the 'Access services' section of the 
+[Get Started guide](https://developers.arcgis.com/java/get-started/#3-access-services-and-content-with-an-api-key) 
+for more information. Help with how to set your API key can be found in the 
+[Developer Guide tutorials](https://developers.arcgis.com/java/maps-2d/tutorials/display-a-map/#set-your-api-key)
+and [Java Samples Repository](https://github.com/Esri/arcgis-runtime-samples-java). If a toolkit component requires an API
+key, this will be indicated within the JavaDoc for the component.
+
+## Instructions
+
+The toolkit library jar is hosted on https://esri.jfrog.io/artifactory/arcgis.
+
+To add the dependency to your project using Gradle:
+```groovy
+plugins {
+    id 'application'
+    id 'org.openjfx.javafxplugin' version '0.0.8'
```

Comment:
if you update to JavaFX 17 this will also need updating

Reply:
Thanks, noted in dependency updates task to be completed.